packages/aws-library/requirements/dev.txt

```diff
@@ -12,7 +12,7 @@
 --requirement _tools.txt
 
 # installs this repo's packages
-pytest-simcore @ ../pytest-simcore
```

Comment:
good catch. Did you check that we did not do elsewhere? :-)

Reply:
no "we" did not ;)